### ISSUE TITLE
Can't get reels videos if title is missing

### ISSUE THREAD

**Issue by wailashraf71** [pgrimaud/instagram-user-feed#309]
```
ErrorException: Trying to get property 'text' of non-object in file C:\xampp\htdocs\app\vendor\pgrimaud\instagram-user-feed\src\Instagram\Hydrator\ReelsHydrator.php on line 24

#0 C:\xampp\htdocs\app\vendor\pgrimaud\instagram-user-feed\src\Instagram\Hydrator\ReelsHydrator.php(24): Illuminate\Foundation\Bootstrap\HandleExceptions-&gt;handleError(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 24, Array)

```

The function of getting reels is working if title is present.

And why there is no reels like or add view to count on views or even comment?

**Comment by pgrimaud** [comment]
This issue has been fixed, please update to `6.15.2`

**Comment by wailashraf71** [comment]
Problem is not solved

**Comment by pgrimaud** [comment]
Could you give me an IG account with this issue?

**Comment by pgrimaud** [comment]
I just fixed it in 6.16, so please upgrade :)
